src/components/Footer.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from "react";
+import "../styles/footer.css";
+import arr from "../data/footer"; /*Importing the footer data file */
+
+class Footer extends Component {
+  render() {
+    return (
+      <div className="footer">
+        {/**main wrapper for the footer */}
+        <div className="footer-logo">
+          {/**main info and links from footer data  */}
+          {arr[0].title}
```

Comment:
can you use arr.map() rather than using arr[<index>]?

Reply:
Yeah I've changed that to map method..gonna commit here in some time...